idea_town/experiments/migrations/0005_auto_20151027_0242.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('experiments', '0004_merge'),
+    ]
+
+    operations = [
+        migrations.AlterField(
```

Comment:
You could probably just lift the params for this `AlterField` and plonk them into the `AddField` for `addon_id` in `0003_auto_20151021_2014.py` and delete these two other migrations as a manual squash. That would muck up your existing dev DB, but should apply cleanly to master